Tests/TableView/TableViewDriverTests.swift

```diff
@@ -213,7 +217,8 @@ final class TableViewDriverTests: XCTestCase {
         let tableView = TestTableView()
         self.setupWithTableView(tableView)
 
-        XCTAssertEqual(tableView.callsToRegisterClass.count, 2)
+        // 1 header + 1 footer + 6 cells
+        XCTAssertEqual(tableView.callsToRegisterClass.count, 8)
```

Comment:
improved count, now that we're exercising the `registerViews(for:)` functionality in the constrained protocol extension